data/gegno/gegno ships.txt

```diff
@@ -0,0 +1,1189 @@
+		"mass" 275
+		"drag" 3.7
+		"heat dissipation" 0.7
+		"fuel capacity" 400
+		"cargo space" 70
+		"outfit space" 285
+		"weapon capacity" 96
+		"engine capacity" 110
+		"weapon capacity" 96
+		weapon
+			"blast radius" 36
+			"shield damage" 210
+			"hull damage" 490
+			"hit force" 470
+	outfits
+		"Chloeric Cannon"
+		"Irate Cannon" 4
+		
+		"C7 Brawl Cell"
+		"R03 Warforge Battery"
+		
+		"RG18 Torch Thruster"
+		"RG18 Torch Steering"
+		"Hyperdrive"
+	engine 0 82
+	gun 0 -98 "Chloeric Cannon"
+	gun -11.5 -53 "Irate Cannon"
+	gun 11.5 -53 "Irate Cannon"
+	gun -11.5 -29 "Irate Cannon"
+	gun 11.5 -29 "Irate Cannon"
+	leak "leak" 60 50
+	leak "flame" 50 80
+	explode "medium explosion" 10
+	explode "small explosion" 20
+	explode "tiny explosion" 15
+	"final explode" "final explosion small"
+	description `While the style of longer battleship-esque vessels is a Gegno Vi staple, another shipyard rose to the competition for Vi war vessels. The Eclogite is one of the newest designs in the Gegno Vi fleet - a gunboat armed to the teeth with cannons, and maneuverability that the Gegno Vi's larger ships lack. It plays the role of a hit-and-run vessel, unloading volleys of shells against a target's armor in a rapid maneuver before adjusting its position for another pounding.`
+
+ship "Diorite"
+	sprite "ship/gegno diorite"
+	thumbnail "thumbnail/gegno diorite"
+	attributes
+		category "Medium Warship"
+		licenses
+			"Vi Centurion"
+		"cost" 6980000
+		"shields" 0
+		"hull" 11600
+		"required crew" 41
+		"bunks" 68
+		"mass" 642
+		"drag" 6.3
+		"heat dissipation" 0.5
+		"fuel capacity" 500
+		"cargo space" 40
+		"outfit space" 548
+		"engine capacity" 124
+		"weapon capacity" 285
+		weapon
+			"blast radius" 90
+			"shield damage" 540
+			"hull damage" 770
+			"hit force" 810
+	outfits
+		"Ballistic Cannon" 2
+		"Chloeric Cannon"
+		"Irate Cannon Turret" 2
+		
+		"C17 Warzone Core"
+		"R03 Warforge Battery"
+		
+		"RG18 Torch Thruster"
+		"RG3 Torch Steering"
+		"Hyperdrive"
+	engine -22.5 158
+	engine 0 164
+	engine 22.5 158
+	gun -7 -193 "Ballistic Cannon"
+	gun 7 -193 "Ballistic Cannon"
+	gun 0 143 "Chloeric Cannon"
+		angle 180
+	turret -24 -77 "Irate Cannon Turret"
+	turret 24 -77 "Irate Cannon Turret"
+	leak "leak" 30 50
+	leak "flame" 50 90
+	explode "large explosion" 9
+	explode "medium explosion" 15
+	explode "small explosion" 20
+	explode "tiny explosion" 17
+	"final explode" "final explosion medium"
+	description "Continuing on with the design concept of the Vi's Eclogite, the Diorite is made by a separate shipyard from other larger Vi vessels. Rather than being a platform for large turreted mounts, the Diorite is focused on having powerful forward weaponry in the form of two mounted Ballistic Cannons and a pair of Irate Cannon Turrets. In conjunction with this configuration, the Diorite is much more maneuverable than other Vi ships in its class and can easily keep a beat on other ships its size, and sometimes smaller ones as well."
```

Comment:
Yes, since the shipyard will be the main shipyard for future Gegno tech ships, I guess it should be named. I'm just worried about the whole, should the player see that once Gegno planets are landable, but perhaps these ships aren't even visible unless you are able to buy them, at which point you'd know the shipyard's name(s).

Reply:
Or perhaps, I can leave this as-is, and then change it in the future when I get to a mission point when I know the description or shipyards would be mentioned, that way I'm more deep into what I'm doing with the lore and missions at that point.

Reply:
We can leave the name up in the air for the moment assuming it actually gets a name in the future.